### DISCUSSION TITLE
Configurable openai `apibase` parameter

### DISCUSSION THREAD

**Discussion by johnd0e** [umputun/tg-spam#134] [Ideas]
Besides OpenAI's there are a lot of other services with compatible API.
Some of them are even free (or have free quota).

**Comment by johnd0e** [comment]
In go-openai it is called `BaseURL`: https://github.com/sashabaranov/go-openai/issues/405#issuecomment-1603914033

**Reply by umputun** [reply]
If you want to submit a PR for this change (it should be trivial, as long as the API is compatible), please do so.

**Reply by johnd0e** [reply]
Well, then https://github.com/umputun/tg-spam/pull/135
